rs_client.py

```diff
@@ -1019,16 +1019,17 @@ def _rs_struct(typeobj):
 
         _r('pub type %s%s = base::StructPtr<%s%s>;', typeobj.rs_type, lifetime1,
                 lifetime2, typeobj.ffi_type)
-        typeobj.struct_ptr = True
 
 
 def _rs_accessors(typeobj):
 
     lifetime = "<'a>" if typeobj.has_lifetime else ""
+    # unions always get an impl lifetime to be sure
+    lifetime2 = "<'a>" if typeobj.is_union else lifetime
 
```

Comment:
Why is that so? Could we not have the lifetime declared at the union accessor directly?
```rust
impl NotifyData {
    pub fn cc<'a>(&'a self) -> CrtcChange<'a> {
        unsafe {
            std::mem::transmute(self)
        }
    }
    ...
}
```